test/unit/index-lambda-cat-bot.test.js

```diff
@@ -1,4 +1,69 @@
-test('this is a placeholder test', () => {
-  console.log('hello, i am running this test!!');
-  expect(true).toBe(true);
+const { handler } = require('../../src/index-lambda-cat-bot');
+const index = require('../../src/index-lambda-cat-bot');
```

Comment:
You can combine lines 1-2! Either by:
```javascript
const {
  handler,
  isTweetingEnabled,
} = require('../../src/index-lambda-cat-bot');

// usage: handler or isTweetingEnabled
```

Or this:
```javascript
const index = require('../../src/index-lambda-cat-bot');

// usage: index.handler or index.isTweetingEnabled
```

Either way works for me, as long as it's consistent 👍